aten/src/ATen/native/LegacyDefinitions.cpp

```diff
@@ -43,21 +44,30 @@ Tensor & masked_scatter__cpu(Tensor& self, const Tensor & mask, const Tensor & s
 }
 
 Tensor masked_select_cpu(const Tensor & self, const Tensor & mask) {
+  Tensor result;
+#ifdef BUILD_NAMEDTENSOR
+  auto outnames = namedinference::compute_broadcast_outnames(self, mask);
+#endif
   if (mask.dtype() == at::ScalarType::Byte) {
     AT_WARN("masked_select received a mask with dtype torch.uint8, this behavior is now deprecated," \
             "please use a mask with dtype torch.bool instead.");
-    return legacy::cpu::_th_masked_select(self, mask);
+    result = legacy::cpu::_th_masked_select(self, mask);
   } else {
-    return legacy::cpu::_th_masked_select_bool(self, mask);
+    result = legacy::cpu::_th_masked_select_bool(self, mask);
   }
+  return result;
 }
 
 Tensor & masked_select_out_cpu(Tensor & result, const Tensor & self, const Tensor & mask) {
+#ifdef BUILD_NAMEDTENSOR
+  namedinference::compute_broadcast_outnames(self, mask);
```

Comment:
this doesn't mutate anything right? is the call here just for error checking?

Reply:
The call is just here for error checking.